src/components/referrals/ReferralsComponent.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+    <div
+        class="referralsBox"
+        @submit.prevent="addTest">
+        <h3 class="referralsBox__header">Dodaj skierowanie:</h3>
+        <form class="referralsBox__form referralsForm">
+            <div class="inputBox">
+                <label for="doctorData"> Lekarz:</label>
+                <select
+                    id="doctorData"
+                    v-model="dataDoctor">
+                    <option
+                        disabled
+                        value="">
+                        Wybierz lekarza
+                    </option>
+                    <option
+                        v-for="research in getDoctorsArray()"
```

Comment:
same as yesterday. Declare array without '()'. Or assign array to new variable and inject here.

Reply:
we need to have (), we talked about that yesterday. i change research to datas

Reply:
Its not good notation. Better when pass data in v-for, not calling getter